docs/content/users/quickstart.md

```diff
@@ -323,7 +323,7 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ```bash
     mkdir my-laravel-app
     cd my-laravel-app
-    ddev config --project-type=laravel --docroot=public
+    ddev config --project-type=laravel --docroot=public --php-version=8.2
     ddev composer create --prefer-dist laravel/laravel -y
```

Comment:
I think we should start to show our expectations on these quickstarts, show the tag they've been tested with. Also `-y` is no longer required with composer create.

```suggestion
    ddev composer create --prefer-dist laravel/laravel:^11
```

Reply:
I was accidentally testing with a `ddev` that wasn't for this version, and got this during composer install:
`SQLSTATE[HY000]: General error: 1273 Unknown collation: 'utf8mb4_0900_ai_ci' (Connection: mysql, SQL: select table_name as `name`, (data_length + index_length) as `size`, table_comment as `comment`, engine as `engine`, table_collation as `collation` from information_schema.tables where table_schema = 'db' and table_type in ('BASE TABLE', 'SYSTEM VERSIONED') order by table_name).`

They seem to default to `mysql:8.0`? 
This is fixed by using `--database=mysql:8.0`. Their system requirements certainly don't hint at this problem, but I'm sure you understand it.

But it means that this quickstart is dependent on using DDEV v1.23.0+, I wish it wasn't.  Maybe we need a tip about it?

Reply:
> Unknown collation: 'utf8mb4_0900_ai_ci'

This is because DDEV sets `DB_CONNECTION=mysql`, but uses MariaDB.

https://github.com/laravel/laravel/blob/11.x/config/database.php#L42-L80

This means when you have  `DB_CONNECTION=mariadb` and MariaDB - it will work fine, but because DDEV v1.22.7 sets it to `mysql`, it complains about a collation.

> But it means that this quickstart is dependent on using DDEV v1.23.0+, I wish it wasn't. Maybe we need a tip about it?

It is already dependent on v1.23.0+ because of

- #5058

But I am open to discussion here (note that I made an updated quickstart for Laravel at the top of the page in #5962).